examples/tls/envoy-https-passthrough.yaml

```diff
@@ -0,0 +1,28 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.tcp_proxy
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
+          cluster: service-https
+          stat_prefix: https_passthrough
```

Comment:
At some point it might be nice to show an example of SNI routing, as this question also comes up a fair amount.

Reply:
see comment above re double-proxy